aidial_analytics_realtime/analytics.py

```diff
@@ -115,6 +135,7 @@ def make_point(
             else uuid4(),
         )
         .field("user_hash", to_string(user_hash))
+        .field("model_price", model_price)
```

Comment:
The Application can be a leaf node too, not only model. I think, the name of the field should be more generic.

Reply:
the field is set by Core for models only. It's not about leaf node. Indeed a leaf node could be an application too.
Any ideas to have better name?

Reply:
as we agreed to remove this field for now